cmd/podman/containers/wait.go

```diff
@@ -54,7 +54,7 @@ func waitFlags(cmd *cobra.Command) {
 	_ = cmd.RegisterFlagCompletionFunc(intervalFlagName, completion.AutocompleteNone)
 
 	conditionFlagName := "condition"
-	flags.StringVar(&waitCondition, conditionFlagName, "stopped", "Condition to wait on")
+	flags.StringSliceVar(&waitConditions, conditionFlagName, []string{}, "Condition to wait on")
```

Comment:
there needs to be a default, later on the check in `libpod/container_api#L:586` needs "stopped" to be propgated if you aren't going to specify `--condition`